templates/js-template-link/frameworks/runtime-src/proj.openharmony/CMakeLists.txt

```diff
@@ -63,7 +68,7 @@ find_library( # Sets the name of the path variable.
               rawfile.z )
 message(STATUS, "rawfile=${rawfile-lib}")
 target_link_libraries(${CC_LIB_NAME} PUBLIC
-${EGL-lib} ${GLESv3-lib} ${hilog-lib} ${libace-lib} ${libnapi-lib} ${libuv-lib} ${rawfile-lib} libnative_drawing.so libc++.a libOpenSLES.so libohaudio.so libavplayer.so
+${EGL-lib} ${GLESv3-lib} ${hilog-lib} ${libace-lib} ${libnapi-lib} ${libuv-lib} ${rawfile-lib} libnative_drawing.so libc++.a libOpenSLES.so libohaudio.so libavplayer.so libjsvm.so libnative_window.so libnative_buffer.so  libbundle_ndk.z.so
```

Comment:
Should libjsvm.so not need to be linked when not using jsvm?

Reply:
If jsvm is not required, there is no need to link libjsvm.so.
We need to optimize here